Configuration/src/main/java/io/deephaven/configuration/PropertyFile.java

```diff
@@ -499,16 +502,107 @@ public Set<String> getNameStringSetFromProperty(final String propertyName) {
         final String propertyValue = getProperty(propertyName).trim();
         switch (propertyValue) {
             case "*":
-                return CollectionUtil.universalSet();
+                // noinspection unchecked
+                return (Set<String>) UNIVERSAL_SET;
             case "":
                 return Collections.emptySet();
             default:
-                Set<String> result = CollectionUtil.setFromArray(propertyValue.split("[, ]+"));
+                String[] data = propertyValue.split("[, ]+");
+                Require.neqNull(data, "data");
+                Set<String> set = new LinkedHashSet<String>((data.length + 1) * 4 / 3);
+                for (String elem : data) {
+                    Require.requirement(set.add(elem), "set.add(elem)");
+                }
+                Set<String> result = set;
```

Comment:
Why do we need two references (`set` and `result`)? What's the point of detecting collisions in this case, but not in `getStringSetFromProperty()`?

Reply:
Both detect collisions, but differently - I had to stare at getStringSetFromProperty longer, so did a slightly better job there.

getStringSetFromProperty does this:

        Require.eq(set.size(), "set.size()", data.length, "data.length");

I can make the error clearer (though it isn't like `set.add(elem)` is a terrible obvious error msg either), but that does the same - if the size of the set is smaller than the array, something was a duplicate.

Will apply the same to (the entirely unused since we axed jgit in 2022) `getNameStringSetFromProperty` method, and inline `result` away.

Reply:
...It _appears_ that the only remaining difference between these methods is that `getStringSetFromProperty` splits on a single `,` or ` ` (space), but `getNameStringSetFromProperty` will split on more than one - that is, their regexes are `[, ]` and `[, ]+` respectively. 

I think we should nuke the bulk of the file rather than get into the weeds with these immortal inconsistencies, but  just wanted to highlight the folly of looking too hard ;).